### PR TITLE
PARQUET-274: Updates URLs to link against the apache user instead of Parquet on github

### PR DESCRIPTION
**Pull request by benpence**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -138,106 +138,106 @@
...
-* ISSUE [397](https://github.com/Parquet/parquet-mr/pull/397): Fixed issue with column pruning when using requested schema
-* ISSUE [389](https://github.com/Parquet/parquet-mr/pull/389): Added padding for requested columns not found in file schema
-* ISSUE [392](https://github.com/Parquet/parquet-mr/pull/392): Value stats fixes
-* ISSUE [338](https://github.com/Parquet/parquet-mr/pull/338): Added statistics to Parquet pages and rowGroups
-* ISSUE [351](https://github.com/Parquet/parquet-mr/pull/351): Fix bug #350, fixed length argument out of order.
-* ISSUE [378](https://github.com/Parquet/parquet-mr/pull/378): configure semver to enforce semantic versioning
-* ISSUE [355](https://github.com/Parquet/parquet-mr/pull/355): Add support for DECIMAL type annotation.
-* ISSUE [336](https://github.com/Parquet/parquet-mr/pull/336): protobuf dependency version changed from 2.4.1 to 2.5.0
-* ISSUE [337](https://github.com/Parquet/parquet-mr/pull/337): issue #324, move ParquetStringInspector to org.apache.hadoop.hive.serde...
+* ISSUE [399](https://github.com/apache/parquet-mr/pull/399): Fixed resetting stats after writePage bug, unit testing of readFooter
```
Comment (isnotinvain) [inline CHANGES.md]: looks like we actually want the old links here

File: README.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ Parquet is a very active project, and new features are being added quickly; belo
 <table>
   <tr><th>Feature</th><th>In trunk</th><th>In dev</th><th>Planned</th><th>Expected release</th></tr>
   <tr><td>Type-specific encoding</td><td>YES</td><td></td></td><td></td><td>1.0</td></tr>
-  <tr><td>Hive integration</td><td>YES (<a href ="https://github.com/Parquet/parquet-mr/pull/28">28</a>)</td><td></td></td><td></td><td>1.0</td></tr>
+  <tr><td>Hive integration</td><td>YES (<a href ="https://github.com/apache/parquet-mr/pull/28">28</a>)</td><td></td></td><td></td><td>1.0</td></tr>
```
Comment (isnotinvain) [inline README.md]: probably not?

**Comment by isnotinvain** [comment]
Can you double check, some of the links should remain as they are and some should get updated.

**Comment by julienledem** [comment]
thanks!

File: README.md [hunk 3]
```diff
@@ -20,9 +20,9 @@
 Parquet MR [![Build Status](https://travis-ci.org/apache/incubator-parquet-mr.svg?branch=master)](http://travis-ci.org/apache/incubator-parquet-mr)
 ======
 
-Parquet-MR contains the java implementation of the [Parquet format](https://github.com/Parquet/parquet-format). 
+Parquet-MR contains the java implementation of the [Parquet format](https://github.com/apache/parquet-format). 
 Parquet is a columnar storage format for Hadoop; it provides efficient storage and encoding of data.
-Parquet uses the [record shredding and assembly algorithm](https://github.com/Parquet/parquet-mr/wiki/The-striping-and-assembly-algorithms-from-the-Dremel-paper) described in the Dremel paper to represent nested structures.
+Parquet uses the [record shredding and assembly algorithm](https://github.com/apache/parquet-mr/wiki/The-striping-and-assembly-algorithms-from-the-Dremel-paper) described in the Dremel paper to represent nested structures.
```
Comment (isnotinvain) [inline README.md]: actually, we don't have the wiki enabled on this repo for some reason, so we need to keep the old link here.

**Pull request closed without merging**